deploy/apache/sites-available/privacyidea.conf

```diff
@@ -55,3 +63,15 @@
 
 
 </VirtualHost>
+
+<VirtualHost _default_:80>
```

Comment:
I would rather not have this port 80 in this file by default.
Can you please remove this?